src/js/modules/Support.js

```diff
@@ -50,16 +50,32 @@ export class Support {
     $('.support-form').each((index, value) => {
       const $form = $(value);
 
-      _this.setAutoValues($form, this.autovalues['show_first']);
+      // Always call setAutoValues with the preferred initial type
+      const showFirst = this.autovalues['show_first'];
+      _this.setAutoValues($form, showFirst);
+
+      // Ensure the first value button is also selected and active
+      const $firstValueBtn = $form
+        .find('.support-form__value-option:visible')
+        .first();
+      if ($firstValueBtn.length) {
+        $firstValueBtn
+          .addClass('ui-button--active')
+          .attr('aria-checked', 'true')
+          .attr('tabindex', '0');
```

Comment:
to understand: why do you set the tab index here programatically?

Reply:
It's to manage keyboard accessibility for the custom radio button group, it mimics native radio buttons. Only the currently selected (active) button is focusable with the Tab key (tabindex="0"). All other buttons are not focusable (tabindex="-1").